classes/cluster/mk20_stacklight_advanced/stacklight/server.yml

```diff
@@ -1,7 +1,7 @@
 classes:
 - system.collectd.client.remote
 - system.elasticsearch.server.cluster
-- system.influxdb.server.cluster
+- system.influxdb.server.single
```

Comment:
Isn't this going to install and start InfluxDB on all the mon nodes? I am not sure, just asking...